### PR TITLE
"Creating Custom Particles" Page

### PR DESCRIPTION
**Pull request by Superkat32**
This PR adds the "Creating Custom Particles" that I wrote for the Community Wiki. It guides the user through the process of registering a new particle with the End Rod Particle's movement using Fabric API. Some feedback is needed, as I didn't know what to do about a few things.

TLDR: Small cleanup from the Community Wiki's page. Need feedback for how to do code snippets in the mod initializer and mod client initializer. I also didn't do anything with the sidebar because there wasn't one for pages about developing yet.

Differences between this page and the page on the Community Wiki:
There's not a lot different to be honest. I mostly made the writing more confident overall. The biggest differences are the following:
- Small clean up on some of the wording of things(specifically the start, going from "Particles are a powerful tool if used correctly" to "Particles are a powerful tool."
- Line highlighting for important and newly added lines of code.
- Removed screenshots of the particle in action and the download link for the particle texture(images can come back if we want them, but I'm not sure how we want to do download links for textures).
- The reference particle's texture has been changed from to the fabric icon(just a random placeholder image I chose, could be the mod icon if we wanted)

Code Snippets:
I decided to manually recreate the code snippets here until I could get some feedback on it. I tried to make it so that I could have a code snippet for the particle registration, but because of the comment about the logger and the logger's "Hello Fabric world!" print out, I didn't know what to do. It seems like you can't specify multiple lines of code to be added to the snippet from the newly merged plugin. 

As for the json code, I also decided to leave that be until feedback was given, as it needs a mod's modid to work, and having the modid as "fabric-docs-reference" in the code snippet might be confusing.

### CODE REVIEW

**Comment by Superkat32** [comment]
I've never heard of a wrong order of imports before, but I'm working on fixing it now.

Edit 1: I think I fixed it. We'll see I guess.

Edit 2: Yep, fixed it. IntelliJ's optimize imports thingy was messing something up.

File: develop/particles/customparticles.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+---
+title: Creating Custom Particles
+description: Learn how to create a custom particle using Fabric API.
+---
+
+# Creating Custom Particles
+
+Particles are a powerful tool. They can add ambience to a beautiful scene, or add tension to an edge of your seat boss battle. Let's add one!
```
Comment (IMB11) [inline develop/particles/customparticles.md:8]:
```suggestion
Particles are a powerful tool. They can add ambience to a beautiful scene, or add tension to your boss battles. Let's add one!
```

**Comment by IMB11** [comment]
This should be ready - migrated to the code snippet system.

**Review by IMB11 — DISMISSED** [review]

**Review by IMB11 — APPROVED** [review]

**Comment by modmuss50** [comment]
Does this need adding to the sidebar?

**Review by modmuss50 — APPROVED** [review]
Looks good, not sure if it needs adding to the sidebar or not in this PR?

**Comment by IMB11** [comment]
Sidebar hasn't been decided yet for develop section, will be in a later PR when more pages are added.

**Pull request merged**